commons/src/main/res/values/integers.xml

```diff
@@ -3,4 +3,6 @@
     <integer name="default_sorting">1</integer>
     <integer name="default_font_size">1</integer>
     <integer name="default_viewpager_page">0</integer>
+    <integer name="contacts_grid_columns_count_portrait">3</integer>
+    <integer name="contacts_grid_columns_count_landscape">5</integer>
```

Comment:
man... the point of this is having different numbers in different resolutions. If you use only have one values/integers.xml, it makes no sense.

Reply:
But that's how it is done in launcher app

Reply:
@tibbi, anyway, what are we going to accomplish by extracting the default column count in resources?

Reply:
not true, the launcher has different values depending on the resolution. For now there are 2 variants only, but more can be added easily.

Reply:
> not true, the launcher has different values depending on the resolution. For now there are 2 variants only, but more can be added easily.

Yeah, I didn't see it in GitHub files, but after importing the project to android studio I was able to see that there are two values. I'll implement the same in commons now. Sorry for the inconvenience.